main.tf

```diff
@@ -71,7 +71,7 @@ module "enhanced_monitoring_iam_role" {
 
 module "db_instance" {
   source                                = "./modules/rds_instance"
-  count                                 = var.create_db_instance ? 1 : 0
+  count                                 = !var.is_cluster ? 1 : 0
```

Comment:
if _not_ true then 1, otherwise 0.

I think that an inverted logic would be easier to read, like:

`var.is_cluster ? 0 : 1`

What do you think?

Reply:
it was to communicate the intention but I have now updated as suggested